tensorflow/contrib/layers/ops/sparse_tile_like_op.cc

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+
+#include "tensorflow/core/framework/common_shape_fns.h"
+#include "tensorflow/core/framework/op.h"
+#include "tensorflow/core/framework/shape_inference.h"
+
+namespace tensorflow {
+
+REGISTER_OP("SparseTileLike")
+    .Input("a_input_indices: int64")
+    .Input("a_input_values: T")
+    .Input("a_input_shape: int64")
+    .Input("b_input_indices: int64")
+    .Input("b_input_values: T")
+    .Input("b_input_shape: int64")
+    .Input("axes: int32")
+    .Output("output_indices: int64")
+    .Output("output_values: T")
+    .Output("output_shape: int64")
+    .Attr("T: realnumbertype")
+    .SetShapeFn(shape_inference::UnknownShape)
+    .Doc(R"doc(
```

Comment:
You now need to put a file in the `api_def` directory to document and generate the python.